android/app/build.gradle

```diff
@@ -33,7 +39,7 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "org.dawnn.dawnnclient"
+        applicationId "org.dawnn.dawnn_client"
```

Comment:
Need to change this to ``org.dawnn.dawnnClient`` to make consistent with iOS - iOS doesn't allow ``-`` in bundle identifiers.

Reply:
This should be done after merge, however. Doesn't relate to FCM notifications IMO.